WORKSPACE

```diff
@@ -1,107 +1,59 @@
 workspace(name="snap_djinni")
 
+load("//bzl:deps.bzl", "djinni_deps")
+djinni_deps()
+
+load("//bzl:setup_deps.bzl", "djinni_setup_deps")
+djinni_setup_deps()
+
+# --- Everything below is only used for examples and tests
+
 android_sdk_repository(name = "androidsdk")
 android_ndk_repository(name = "androidndk", api_level = 21)
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
-load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
-# Bazel Skylib needed for Scala rules
+build_bazel_rules_apple_version = "0.31.3"
```

Comment:
I've moved all of the android and iOS deps back to WORKSPACE since we don't need it in the dependencies exposed externally